Build/Projects/MonoGame.Framework.definition

```diff
@@ -292,7 +292,7 @@
     <Compile Include="DisplayOrientation.cs" />
     <Compile Include="DrawableGameComponent.cs" />
     <Compile Include="FrameworkDispatcher.cs">
-      <Platforms>_FrameworkDispatcherProvided</Platforms>
+      <Services>_FrameworkDispatcherProvided</Services>
```

Comment:
This is starting to maybe get deeper than your fix here @Jjagg .

But this looks like a big problem for the unit tests:

  https://github.com/mono/MonoGame/blob/develop/MonoGame.Framework/FrameworkDispatcher.cs#L12

We should move the sound updates from `Game` into there:

https://github.com/mono/MonoGame/blob/develop/MonoGame.Framework/Game.cs#L653

Then call `FrameworkDispatcher.Update()` from `Game` instead.